### ISSUE TITLE
Missing "file_copy" icon in Material Icons Font causes horizontal overflow

### ISSUE THREAD

**Issue by tomayac** [GoogleChrome/web.dev#3768]
The "Copy code" button for code blocks in articles references an apparently missing icon `"file_copy"` (I assume too aggressive font subsetting, since the glyph is only needed on hover), which causes a horizontal overflow. See the bottom bar with horizontal overflow and the non-working icon:

<img width="742" alt="Screen Shot 2020-08-24 at 15 37 37" src="https://user-images.githubusercontent.com/145676/91051076-e0e99300-e61f-11ea-8de3-1fe8cbcd9b2f.png">

The code for the button is this (the icon is specified via `data-icon`):

```html
<button class="w-button--icon w-button--round web-copy-code__button" data-icon="file_copy" aria-label="Copy code">
  <!-- -->
</button>
```

Manually overriding the `data-icon`with a value of `close` (which does exist), works. See the bottom bar without horizontal overflow and the working icon"

<img width="728" alt="Screen Shot 2020-08-24 at 15 37 10" src="https://user-images.githubusercontent.com/145676/91051072-ddeea280-e61f-11ea-9f10-e19163736099.png">

(Reported [here](http://twitter.com/simevidas/status/1297686248396656640) by @simevidas).

**Comment by robdodson** [comment]
Probably related to this change https://github.com/GoogleChrome/web.dev/pull/3755

**Comment by simevidas** [comment]
A little observation. Removing `word-wrap: normal` from the `.w-button--icon::before` fixes the issue, but it also shifts the icon off center.

![giphy](https://user-images.githubusercontent.com/716405/91076274-8a8c4c80-e63f-11ea-98ed-199237679a5d.gif)

**Comment by robdodson** [comment]
Should be fixed by #3798 

Good catch on the word-wrap issues @simevidas. I'll see if I can add that in a follow up PR
